kaldo/helpers/typing.py

```diff
@@ -0,0 +1,9 @@
+from numpy.typing import NDArray, ArrayLike
+from os import PathLike
+from ase import Atoms
+
+NDArray = NDArray
```

Comment:
Better not to create a `typing` helper file. We can use the native references to `NDArray`, `ArrayLike`, `Atoms` ...  as needed.

Reply:
cc @bkhli